pipelines/rj_crm__get_history_data/flow.py

```
@@ -0,0 +1,93 @@
+                de_name=de["name"],
+                soap_uri=soap_uri,
+            )
+            extraction_results.append({
+                "de_name": de["name"],
+                "external_key": de["external_key"],
+                "status": "success",
+                "total_rows": len(rows),
+                "dados": rows,
+                "tipos": field_types,
+            })
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)